go.mod

```diff
@@ -20,6 +20,7 @@ require (
 require (
 	github.com/Rican7/retry v0.3.1 // indirect
 	github.com/armon/go-proxyproto v0.1.0 // indirect
+	github.com/canonical/go-dqlite v1.22.0 // indirect
```

Comment:
@roosterfish what is left that is depending on go-dqlite v1?

Reply:
I am pretty sure that comes from the old version of LXD before merging the dqlite v2 PR.

Reply:
Should be fixed after we track `@stable-5.21`.

Reply:
But you're not changing this branch to track `@stable-5.21` so should already be gone by updating the lxd dep?

Reply:
The LXD dep used here was still an older one before LXD was switched to dqlite v2.